### PR TITLE
Add test for extract_vfolder_from_path function

### PR DESCRIPTION
**Pull request by unho**
Followup for #4316.

### CODE REVIEW

File: tests/virtualfolder_helpers.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
```
Comment (phlax) [inline tests/virtualfolder_helpers.py]: this line is unnecessary

Comment (unho) [inline tests/virtualfolder_helpers.py]: Perhaps, but I just copied this from other file in tests. I checked 5 of them randomly and they all have this line. Doesn't harm anyway.

File: tests/virtualfolder_helpers.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from __future__ import absolute_import
```
Comment (phlax) [inline tests/virtualfolder_helpers.py]: i dont think this is necessary either

**Comment by phlax** [comment]
nits aside - lgtm - nice one!

File: tests/virtualfolder_helpers.py (hunk 1 above)
Comment (phlax) [inline tests/virtualfolder_helpers.py]: yep - but no need to copy out bad code

**Comment by phlax** [comment]
lgtm

**Pull request merged**